### PR TITLE
Fix function indicator bar on output port

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

Task: https://jira.autodesk.com/browse/DYN-5017

When the CachedValue is not a function, PortDefaultValueMarkerVisible should be set to false.

function indicator fix

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes
Fix function indicator bar on output port

### Reviewers
@QilongTang .

### CODE REVIEW

**Comment by QilongTang** [comment]
@filipeotero When you have time, please review this

**Review by filipeotero — APPROVED** [review]
LGTM

File: src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs [hunk 1]
```diff
@@ -309,16 +309,7 @@ private void OnMouseLeftButtonDownOnContext(object parameter)
...
-
-            if (node.NodeModel.CachedValue != null  && node.NodeModel.CachedValue.IsFunction)
-            {
-                PortDefaultValueMarkerVisible = true;
-            }
-            else if (isCachedValueNull)
-            {
-                PortDefaultValueMarkerVisible = false;
-            }
+            PortDefaultValueMarkerVisible = node.NodeModel.CachedValue != null && node.NodeModel.CachedValue.IsFunction;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs:312]: Do we still need to check `node.NodeModel.CachedValue.Data == null`?

Comment (reddyashish) [reply src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs:312]: when node.NodeModel.CachedValue.Data is null, IsFunction will be false and the port marker will not be visible.

**Comment by reddyashish** [comment]
@QilongTang Removing those unused images that were left out from https://github.com/DynamoDS/Dynamo/pull/13109. Sorry I mixed up branches and just consolidated it.

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@reddyashish Looks like there are regressions to fix

**Comment by reddyashish** [comment]
This test WorkspaceContextMenu_IfSubmenuOpenOnMouseHover passes locally and passed on CI before. It just failed for some other reason. Looks flaky. Verified it locally.

**Comment by reddyashish** [comment]
Running it again: https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/6515/

**Comment by reddyashish** [comment]
Lastest CI job passed

**Review by QilongTang — APPROVED** [review]

**Pull request merged**